nixos/modules/services/web-apps/peering-manager.nix

```
@@ -172,8 +192,16 @@ in {
         with open("${cfg.peeringdbApiKeyFile}", "r") as file:
           PEERINGDB_API_KEY = file.readline()
       '';
-
-      plugins = lib.mkIf cfg.enableLdap (ps: [ ps.django-auth-ldap ]);
+      # TODO think about adding a oidcSecretFile option to use secrets
+      # + lib.optionalString (cfg.oidcSecretFile != null) ''
+      #  with open("${cfg.oidcSecretFile}", "r") as file:
+      #    OIDC_RP_CLIENT_SECRET = file.readline()
+      #'';
+
+      plugins = lib.mkIf (cfg.enableLdap || cfg.enableOidc) (ps: [
+        (lib.mkIf cfg.enableLdap ps.django-auth-ldap)
+        (lib.mkIf cfg.enableOidc ps.mozilla-django-oidc)
+      ]);
```

This should fix your OpenSSL issue:
```suggestion
      plugins = lib.mkIf (cfg.enableLdap || cfg.enableOidc) (ps: [
        (lib.mkIf cfg.enableLdap ps.django-auth-ldap)
        (lib.mkIf cfg.enableOidc ps.mozilla-django-oidc)
        (lib.mkIf cfg.enableOidc ps.pyopenssl)
      ]);
```

---

At least the import error is gone; I need to test if oidc is actually working